lib/Interpreter/CIFactory.cpp

```diff
@@ -928,29 +961,132 @@ namespace {
   }
 
   static void AddRuntimeIncludePaths(llvm::StringRef ClingBin,
-                                     clang::HeaderSearchOptions& HOpts) {
-    if (HOpts.Verbose)
-      llvm::errs() << "Adding runtime include paths:\n";
-    // Add configuration paths to interpreter's include files.
+                                    std::vector<const char*>& args,
+                                    bool Verbose) {
+    static std::vector<std::string> sArguments;
```

Comment:
I generally agree, but was following the pattern of AddHostArguments.
I thought it did statics for two reasons:
 Avoid redoing work.
 Make sure all children get the same flags as parent. In an interpreted environment there's no guarantee that would be true across calls if they are not static.